ice_setup/ice.py

```diff
@@ -1174,7 +1214,7 @@ def configure_ceph_deploy(master, minion_url, minion_gpg_url,
     # ensure we write the config file in all these places because the $HOME
     # location might not be what the user expected to be
     configs = [
-        os.path.join(os.getcwd(), 'cephdeploy.conf'),
+        os.path.join(CWD, 'cephdeploy.conf'),
```

Comment:
Good point.  I believe the goal here is to write three places:

 /opt/ICE/cephdeploy.conf
~/.cephdeploy.conf
/root/.cephdeploy.conf  (the last two may be the same, depending on sudo).

The previous instructions were to always run the script from /opt/ICE, so using CWD worked.  Sounds like this should change to /opt/ICE instead of CWD.  Do you agree?